src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -183,17 +183,31 @@ private int TryGetAppSettingAsIntWithDefault(string key, int defaultValue)
 
         public object AgentRunId { get { return _serverConfiguration.AgentRunId; } }
 
+        private bool _agentEnabled;
```

Comment:
We might not need this field anymore because we are not using its value to short-circuit any logic.